recipes/rkstiff/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "rkstiff" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9d17c39e7c2a2cf8210fc563bc834eb4bd5513969f1664c4cdfeb2d0571fe6a4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6 
+    - pip
+    - numpy >=1.14.0
+    - scipy >=1.3.2
```

Comment:
Do they need to be in build?

Reply:
Hi xhochy, no, they are not needed in build.  Nothing is compiled in this version.

Reply:
I put numpy and scipy under host requirements

Reply:
Not sure if they are needed under host. I used the conda_build skeleton command to generate meta.yml from the pypi package source and they were under host requirements so I left them there.

Reply:
Try to leave them out of it, normally they are not required.

Reply:
Giving it a go

Reply:
Hmmm. The checks did not pass the build tests. This seems to be the culprit
ERROR: No matching distribution found for scipy>=1.3.2
I think I need to do a bit of digging as to why the conda skeleton pypi command is putting these things into the host section, maybe I have a weird setting in setup.py.

Reply:
Using https://github.com/conda-incubator/grayskull instead of `conda skeleton` will generally produce better recipes. You could give that a try.

Reply:
Thanks for the tip with grayskull, it seems to generate a cleaner meta.yaml file requiring fewer edits on my end. Unfortunately it still adds scipy and numpy to the host requirements section. I'm going to try removing the version requirement from scipy and see if I'm still having test build issues.

Reply:
Weird setting in setup.py confirmed, I had scipy in the setup_requires setting. Thats why the package only would build with scipy under the host requirements. Loaded a new version of rkstiff onto PyPI with the amended setup.py and ran it through the conda-forge tests.